data/sql/grid_transformation_custom.sql

```diff
@@ -147,3 +147,25 @@ INSERT INTO "usage" VALUES(
     'EPSG','1120', -- area of use: Iceland - onshore and offshore
     'EPSG','1024'  -- unknown
 );
+
+-- Japan
+
+INSERT INTO "grid_transformation" VALUES(
+    'PROJ','EPSG_7906_TO_EPSG_6695','ITRF94 to JGD2011 height',
```

Comment:
there's an inconsistency here.   'PROJ','EPSG_7906_TO_EPSG_6695','ITRF94 to JGD2011 height' is OK , but this is not consistent with     'EPSG','6667', -- source CRS (JDG2011)
You need to decide if the source CRS is ITRF94 or JGD2011 and adjust things accordingly

Reply:
I have to verify the grid (in fact I was wondering why I put ITRF94 there in the first place).

Reply:
It will be JDG2011. I assume that I didn't a mistake in that regard when I created the grid.